ReactWindows/ReactNative.Tests/Modules/Clipboard/ClipboardModuleTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
+using ReactNative.Bridge;
+using ReactNative.Modules.Clipboard;
+using System;
+
+namespace ReactNative.Tests.Modules.Clipboard
+{
+    [TestClass]
+    public class ClipboardModuleTests
+    {
+        [TestMethod]
+        public void ClipboardModule_ArgumentChecks()
+        {
+            var module = new ClipboardModule(new ReactContext());
+
+            AssertEx.Throws<ArgumentNullException>(
+                () => module.getString(null),
+                ex => Assert.AreEqual("promise", ex.ParamName));
+        }
+
+        [TestMethod]
+        public void ClipboardModule_GetString_Method()

```

Comment:
Just for safety, can we ensure that this test method blocks until the promise is resolved, and then asserts on the result?  I don't think this test can actually fail as the Assert.AreEqual occurs on a different thread.